aws-lc-rs/src/pbkdf2.rs

```diff
@@ -168,8 +168,13 @@ const MAX_USIZE32: u64 = u32::MAX as u64;
 /// algorithm's output length, per the PBKDF2 specification.
 ///
 /// # FIPS
-/// FIPS users should utilize this method when meeting the following conditions:
-/// * Algorithms: `HMAC_SHA1_FOR_LEGACY_USE_ONLY`, `HMAC_SHA224`, `HMAC_SHA256`, `HMAC_SHA384`, or `HMAC_SHA512`.
+/// The following conditions must be met:
+/// * Algorithm is one of the following:
+///   * `HMAC_SHA1_FOR_LEGACY_USE_ONLY`
+///   * `HMAC_SHA224`
```

Comment:
1. It seems like we're missing a `PBKDF2_HMAC_SHA224`, so this might not be applicable :(
2. Should we change these to point to the PBKDF2 constructs instead?
```
/// * `PBKDF2_HMAC_SHA1`
/// * `PBKDF2_HMAC_SHA256`
/// * `PBKDF2_HMAC_SHA384`
/// * `PBKDF2_HMAC_SHA512`
```

Reply:
Good catch!